TarSCM/scm/git.py

```diff
@@ -122,42 +123,17 @@ def detect_version(self, args):
             versionformat = '%ct.%h'
 
         if not parent_tag:
-            cmd = ['git', 'describe', '--tags', '--abbrev=0']
-            try:
-                if args['match_tag']:
-                    cmd.append("--match=%s" % args['match_tag'])
-            except KeyError:
-                pass
-            rc, output = self.helpers.run_cmd(cmd, self.clone_dir)
-
-            if rc == 0:
-                # strip to remove newlines
-                parent_tag = output.strip()
+            parent_tag = self._detect_parent_tag(args)
```

Comment:
This commit seems to have some refactorings or other parent_tag related changes which are unrelated to linting cleanups - is that deliberate?  Not sure what those changes are for, so I can't really review them.